osu.Game/Beatmaps/Drawables/BeatmapSetDownloader.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Configuration;
+using osu.Framework.Graphics;
+
+namespace osu.Game.Beatmaps.Drawables
+{
+    public class BeatmapSetDownloader : Drawable
```

Comment:
It is more correct to derive from `Component` here
https://github.com/ppy/osu-framework/blob/master/osu.Framework/Graphics/Component.cs